### PR TITLE
Feat/footer update

### PR DESCRIPTION
**Pull request by nym1k**
- Update structure of main site footer
- Remove old footer navigation and partners sections
- Add new partners logos
- Add new footer logo
- Update footer bottom to include new terms & conditions page link, and remove social link

### CODE REVIEW

File: app/assets/stylesheets/components/footer.scss [hunk 1]
```diff
@@ -1,190 +1,154 @@
...
-    @include only-print {
-      width: 80%;
-    }
   }
 
-  @include e(navigation) {
-    background-color: $prussian-blue;
-    line-height: vr(2);
+  @include e(items) {
+    @media (min-width: $break__desktop) {
```
Comment (WillKocur) [inline app/assets/stylesheets/components/footer.scss:22]: Is this the same as only_desktop()?

Comment (nym1k) [reply app/assets/stylesheets/components/footer.scss:22]: No, this is a new breakpoint I had to include for the header and footer.  (`$break__desktop: 1280px;`)

The mixin only_desktop works like this:

`@mixin only-desktop {
  @media (min-width: $break__phone) {
    @content;
  }
}`

File: app/views/layouts/application/_footer.html.erb [hunk 2]
```diff
@@ -1,82 +1,99 @@
...
+  <div class="footer__footer">
+    <div class="footer__inner">
+      <div class="footer__copyright">
+        <div class="footer__copyright-items">
+          <div class="footer__copyright-item">
+            <a
+              href="https://www.unep-wcmc.org"
+              class="footer__copyright-link"
+            >
+              &copy; <%= Date.today.year %> UNEP WCMC
```
[the remark this answers is not in the document]
Comment (nym1k) [reply app/views/layouts/application/_footer.html.erb:84]: That's how it was originally 😅

File: app/assets/stylesheets/components/footer.scss (hunk 1 above)
Comment (WillKocur) [reply app/assets/stylesheets/components/footer.scss:22]: Oh I see, sorry should have checked!

**Comment by WillKocur** [comment]
@nym1k Happy for this to be merged - will leave for you to do when ready.

**Pull request merged**